### ISSUE TITLE
Show NACs on the map as images rather that just footprints

### ISSUE THREAD

**Issue by foobarbecue** [nasa-jpl/sstmp#7]
Right now, we show nac footprints but no actual imagery.

We should load thumbnail images when we first add the footprints. We could also create a tile layer after or during the cam2map step and add that to the map.

**Comment by thareUSGS** [comment]
If you were to create the final image as COGs (Cloud Optimized GeoTIFFs) then you might be able to support this using a Lambda AWS machine and Rio-tiler: https://cogeotiff.github.io/rio-tiler/advanced/dynamic_tiler/  or https://github.com/developmentseed/cogeo-tiler . We are hoping to implement this in front of our individual STAC/COGs releases for others to use.

**Comment by foobarbecue** [comment]
Yep, I was thinking COG would be the way to go for this. I hadn't looked into the details yet so I appreciate these pointers.
